src/Uno.UI/UI/Xaml/Controls/ToolTip/ToolTip.cs

```diff
@@ -73,10 +129,600 @@ private void AttachToPopup()
 			}
 		}
 
-		public void SetAnchor(UIElement element) => _popup.Anchor = element;
+		public void SetAnchor(UIElement element) => _owner = element;
 
 		public event RoutedEventHandler Closed;
 
 		public event RoutedEventHandler Opened;
+
+		// Sets the location of the ToolTip's Popup.
+		//
+		// Slider "Disambiguation UI" ToolTips need special handling since they need to remain centered
+		// over the sliding Thumb, which has not yet rendered in its new position.  Therefore, we pass
+		// the new target rect to handle this case.
+		private void PerformPlacement(Rect? pTargetRect = null)
+		{
+			// It is possible for this function to be called even though the ToolTip is closed.
+			// This can happen if the ToolTip gets closed before it has had a chance to layout and complete its opening sequence.
+			// If this happens, we don't want to continue opening, as that could result in a "closed" ToolTip that is still visible on the screen.
+			var isOpen = IsOpen;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'isOpen'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5404644)